python/lsst/ts/electrometer/commander.py

```diff
@@ -69,20 +69,33 @@ def __init__(self, log: None | logging.Logger = None) -> None:
         self.lock: asyncio.Lock = asyncio.Lock()
         self.host: str = tcpip.LOCAL_HOST
         self.port: int = 9999
-        self.timeout: int = 2
+        self.timeout: int = 5
         self.long_timeout: int = 30
         self.connected: bool = False
+        self.brand: str = None
 
     async def connect(self) -> None:
         """Connect to the electrometer"""
         async with self.lock:
             try:
                 connect_task = asyncio.open_connection(
-                    host=self.host, port=int(self.port), limit=1024 * 1024
+                    host=self.host, port=int(self.port), limit=1024 * 1024 * 10
                 )
                 self.reader, self.writer = await asyncio.wait_for(
                     connect_task, timeout=self.long_timeout
                 )
+                # get the welcome message
```

Comment:
I think that we should have the brand information come from the configuration and that determines what message to look for and handle the setup appropriately. Also if the welcome message is incorrect i.e. it's a keithley and not a keysight or it's a keysight and not a keithley. We should error out. Also the exception is too broad as you could have no response with a time out.